website/static/js/profile.js

```diff
@@ -675,24 +763,55 @@ ListViewModel.prototype.unserialize = function(data) {
 
 ListViewModel.prototype.serialize = function() {
     var contents = [];
-    if (this.contents().length !== 0 && typeof(this.contents()[0].serialize() !== undefined)) {
+    if (this.contents().length !== 0 && typeof(this.contents()[0].serialize()) !== 'undefined') {
         for (var i=0; i < this.contents().length; i++) {
-            contents.push(this.contents()[i].serialize());
+            // If the requiredField is empty, it will not save it and will delete the blank structure from the database.
+            if (!( this.contents()[i].institutionEmpty() && this.hasMultiple() )) {
+                contents.push(this.contents()[i].serialize());
+            }
+            else  {
+                this.contents.splice(i, 1);
+            }
         }
     }
     else {
         contents = ko.toJS(this.contents);
     }
-
     return {contents: contents};
 };
 
+ListViewModel.prototype.submit = function() {
```

Comment:
This override should not be necessary; `self.isValid()` should take care of any validation before submitting the form.